webknossos/pyproject.toml

```diff
@@ -29,29 +31,29 @@ classifiers = [
 include = ["webknossos/version.py"]  # included explicitly since it's in .gitignore
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<3.10"
+python = ">=3.8,<3.11"
 aiohttp = "^3.8.1"
-attrs = "^21.1.0"
+attrs = "^22.1.0"
 boltons = "~21.0.0"
-cattrs = ">=1.7.1"
+cattrs = "^22.2.0"
 cluster_tools = { path = "../cluster_tools/", develop = true }
 fsspec = "^2022.2.0"
 httpx = ">=0.15.4,<0.19.0"
 loxun = "^2.0"
 natsort = "^6.2.0"
 networkx = "^2.6.2"
 numcodecs = "^0.10"
-numpy = "^1.15.0"  # see https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table
+numpy = "^1.21.0"  # see https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table
```

Comment:
We can now replace our custom fallback `DTypeLike` definition in `webknossos/dataset/layer.py`, since `numpy.typing.DTypeLike` is available since numpy 1.20